bin/ch-image.py.in

```diff
@@ -177,6 +177,8 @@ def main():
    sp.set_defaults(func=push.main)
    add_opts(sp, common_opts, True)
    sp.add_argument("source_ref", metavar="IMAGE_REF", help="image to push")
+   sp.add_argument("--hash-only", action="store_true",
+                   help="stop after obtaining compressed layer hash")
```

Comment:
How about `--prepare-only`? And possibly "stop after preparing; don't upload or clean up temp files"?